core/src/test/scala/sangria/federation/v2/Link__Import_Spec.scala

```diff
@@ -0,0 +1,24 @@
+package sangria.federation.v2
+
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.EitherValues
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class Link__Import_Spec extends AnyWordSpec with Matchers with MockFactory with EitherValues {
+
+  "Scalar Link__Import coercion (https://specs.apollo.dev/link/v1.0/#example-import-a-string-name)" should {
+    "accept a string" in {
+      Link__Import.Type.coerceUserInput("str").value shouldBe a[Link__Import]
+    }
+    "accept an object with 'name' and an optional 'as' string fields" in {
+      Link__Import.Type
+        .coerceUserInput(Link__Import_Object("str"))
+        .value shouldBe a[Link__Import_Object]
+    }
+    "raise violation error in any other case" in {
+      Link__Import.Type.coerceUserInput(()).left.value should be(
+        Link__Import.Link__Import_Coercion_Violation)
+    }
```

Comment:
"accept an object with 'name' and an optional 'as' string fields" need another case with a value for `as`.

```suggestion
  "Scalar Link__Import coercion" should {
    "accept a string" in {
      // https://specs.apollo.dev/link/v1.0/#example-import-a-string-name
      Link__Import.Type.coerceUserInput("str").value shouldBe a[Link__Import]
    }
    "accept an object with 'name' and an optional 'as' string fields" in {
      // https://specs.apollo.dev/link/v1.0/#example-import-an-aliased-name
      Link__Import.Type
        .coerceUserInput(Link__Import_Object("str"))
        .value shouldBe a[Link__Import_Object]
    }
    "raise violation error in any other case" in {
      // https://specs.apollo.dev/link/v1.0/#example-incorrectly-importing-a-transitive-schema-reference
      Link__Import.Type.coerceUserInput(()).left.value should be(
        Link__Import.Link__Import_Coercion_Violation)
    }
```

Reply:
actually, we do not implement https://specs.apollo.dev/link/v1.0/#example-incorrectly-importing-a-transitive-schema-reference for now. The comment can be removed.